addons/account/models/account_journal.py

```diff
@@ -970,6 +960,17 @@ def _get_journal_bank_account_balance(self, domain=None):
         journal_currency = self.currency_id if self.currency_id and self.currency_id != company_currency else False
         return amount_currency if journal_currency else balance, nb_lines
 
+    def _has_dedicated_payment_sequence(self):
+        self.ensure_one()
+
+        return (
+            self.type in ['cash', 'bank']
+            and (
+                any(self.outbound_payment_method_line_ids.mapped('payment_sequence'))
+                or any(self.inbound_payment_method_line_ids.mapped('payment_sequence'))
+            )
+        )
```

Comment:
So activating it on one line means that it's activated on the journal altogether? So on all the lines?